code/datums/components/bakeable.dm

```diff
@@ -93,11 +93,11 @@
 	var/list/asomnia_hadders = list()
 	for(var/mob/smeller in get_hearers_in_view(DEFAULT_MESSAGE_RANGE, used_oven))
 		if(HAS_TRAIT(smeller, TRAIT_ANOSMIA))
-			asomnia_hadders += smeller 
+			asomnia_hadders += smeller
 
 	if(positive_result)
 		used_oven.visible_message(
-			span_notice("You smell something great coming from [used_oven]."), 
+			span_notice("You smell something great coming from [used_oven]."),
```

Comment:
bloon alert this one

Reply:
From experience, this isn't something you want to miss out in the midst of preparing several dishes lest you burn the contents to a crisp, and the visible message is much more helpful than a balloon message.